src/NHSD.BuyingCatalogue.Organisations.Api/Controllers/OrganisationsController.cs

```diff
@@ -261,5 +261,31 @@ public async Task<ActionResult> CreateRelatedOrganisationAsync(Guid id, [FromBod
 
             return NoContent();
         }
+
+        [Authorize(Policy = PolicyName.CanAccessOrganisations)]
+        [HttpDelete]
+        [Route("{organisationId}/related-organisations/{relatedOrganisationId}")]
+        public async Task<ActionResult> DeleteRelatedOrganisationAsync(Guid organisationId, Guid relatedOrganisationId)
+        {
+            var organisation = await organisationRepository.GetByIdWithRelatedOrganisationsAsync(organisationId);
+
+            if (organisation is null)
+            {
+                return NotFound();
+            }
+
+            if (!organisation.RelatedOrganisations.Any(ro => ro.OrganisationId == relatedOrganisationId))
+            {
+                return BadRequest(new ErrorMessageViewModel(FormattableString.Invariant($"The referenced organisation {organisationId} has no relationship to {relatedOrganisationId}.")));
+            }
+
+            var relatedOrganisation = organisation.RelatedOrganisations.Where(ro => ro.OrganisationId == relatedOrganisationId).First();
+
+            organisation.RelatedOrganisations.Remove(relatedOrganisation);
+
+            await organisationRepository.UpdateAsync(organisation);
```

Comment:
I think we could do something like this here:

```c#
if (organisation.RelatedOrganisations.Remove(relatedOrganisation))
    await organisationRepository.UpdateAsync(organisation);
```
This way we will only update if the related orgs have actually changed and we return `NoContent()` in both cases where we've deleted the relationship and where the relationship didn't exist – I don't think we need to differentiate.